### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by purvasharmadev**
## major changes

> added login feature with a guest login option
> added signup feature

### changes to ignore

> _redirect file
> style changes

### CODE REVIEW

File: src/Components/auth/signup.jsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { Link } from "react-router-dom";
+import { Footer } from "../../Pages/Footer";
+import { Nav } from "../../Pages/Nav";
+import { useSignUp } from "../../Hooks/useSignupUser";
+
+function SignUp() {
+  const { sucess, error, setUserData, userData, postUserSignIn } = useSignUp();
+  function signUpHandler(event) {
+    console.log("signUphandler running........");
```
Comment (iRohitGaur) [inline src/Components/auth/signup.jsx:9]: remove `console.log`

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

File: src/Components/auth/login.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import { Nav } from "../../Pages/Nav";
+import { useLogin } from "../../Hooks/useLoginUser";
+
+function Login() {
+
+  const {sucess,error,setUserData,userData, postUserLogin} = useLogin()
+
+  const guestLogin = () => {
+    console.log("guestLogin running...")
+    setUserData((prev) => ({
```
Comment (srishtigautam21) [inline src/Components/auth/login.jsx:12]: you can remove the console

**Review by srishtigautam21 — APPROVED** [review]
well structured code
Good to merge

**Review by adityapandadev — APPROVED** [review]

**Comment by adityapandadev** [comment]
Good to merge 👍

**Pull request merged**